src/shared/containers/LibraryCardForm/LibraryCardForm.jsx

```diff
@@ -297,7 +297,7 @@ class LibraryCardForm extends React.Component {
           formEntrySuccessful: false,
           apiResults: response.data,
         });
-        window.location.href = `${config.confirmationURL.base}?patronID=${response.data.response.patron_id}&patronName=${this.getFullName()}`;
```

Comment:
I think we use the queries to get the name for our `Confirmation.jsx`. Deleting it might result in the blank value.

Reply:
@ktp242 I think `Confirmation.jsx` is no longer in use. We choose to just redirect to Petrol so that the page could be edited via CMS.